third-party/libfabric/Makefile

```diff
@@ -17,8 +17,8 @@ endif
 CHPL_LIBFABRIC_CFG_OPTIONS += --enable-static \
                               --disable-shared \
                               --prefix=$(LIBFABRIC_INSTALL_DIR) \
-			      --disable-psm3 \
-			      --disable-opx
+                              --disable-psm3 \
+                              --disable-opx
```

Comment:
Minor, but let's not include unrelated changes in this already large PR

Reply:
Undone.